server/net_utils/tcp_server.c

```diff
@@ -19,10 +19,15 @@ static void bind_and_listen_on_socket(tcp_server_t *srv, long port)
     srv->self.sin_port = htons(port);
     srv->self.sin_addr.s_addr = htonl(INADDR_ANY);
     if (bind(srv->sock_fd, (struct sockaddr *) &srv->self,
-        sizeof(struct sockaddr)) < 0)
+        sizeof(struct sockaddr)) < 0){
         TEAMS_LOG("bind");
-    if (listen(srv->sock_fd, LISTEN_BACKLOG) < 0)
+        return false;
+    }
+    if (listen(srv->sock_fd, LISTEN_BACKLOG) < 0) {
         TEAMS_LOG("listen");
+        return false;
+    }
+    return true;
 }
 
 bool add_user_to_server(tcp_server_t *srv, char *username, char *password)
```

Comment:
Add user to server ? Function seems to be server init no ?

Reply:
This function will not be used since it's proper to the net_utils and not available in outer scope

Reply:
My bad, I don't know how to read a file correctly...